minifi_main/CMakeLists.txt

```
@@ -52,8 +52,9 @@ endif(NOT USE_SHARED_LIBS)
 target_link_libraries(minifiexe Threads::Threads)
 
 target_link_libraries(minifiexe yaml-cpp)
-if(NOT WIN32 AND ENABLE_JNI AND NOT DISABLE_JEMALLOC)
-    target_link_libraries(minifiexe JeMalloc::JeMalloc)
+if(CUSTOM_MALLOC_LIB)
+    message("Using custom malloc lib ${CUSTOM_MALLOC_LIB} for minifiexe")
```

I'd lower the log level here as well, we don't need to print setting values during normal usage.
```suggestion
    message(VERBOSE "Using custom malloc lib ${CUSTOM_MALLOC_LIB} for minifiexe")
```

---

:+1: https://github.com/apache/nifi-minifi-cpp/pull/1456/commits/d08d3c78709c3a1042429de7ac18591909f6e4d2#diff-b8c99bced444e4532cfed3253588140a9cb21b8f94d4fdd0e4cbbdea8a855789R56

---

:+1: https://github.com/apache/nifi-minifi-cpp/pull/1456/commits/d08d3c78709c3a1042429de7ac18591909f6e4d2#diff-b8c99bced444e4532cfed3253588140a9cb21b8f94d4fdd0e4cbbdea8a855789R56